pdfrw/tokens.py

```diff
@@ -95,24 +83,23 @@ def _gettoks(self, startloc, cacheobj=_cacheobj,
         fdata = self.fdata
         current = self.current = [(startloc, startloc)]
         cache = {}
+        get_cache = cache.get
         while 1:
             for match in findtok(fdata, current[0][1]):
                 current[0] = tokspan = match.span()
                 token = match.group(1)
                 firstch = token[0]
+                toktype = intern
```

Comment:
What can I say?  C is the new assembly; Python is the new C.  And I don't even drop down to the new assembly unless absolutely required :-)

Thanks for looking it over.